src/main/external-resources/documentation/introduction.html

```diff
@@ -152,19 +152,19 @@
 	<p>You can stream media files with quite old computers. If the
 		device does not recognize the video container format and UMS has to
 		remux you will need some additional CPU power. A 2 GHz CPU should be
-		sufficient. When transcoding a video on full HD resolution an up to
-		date dual core processor from Intel or AMD with at least 2,6 GHz is
+		sufficient. When transcoding a full HD resolution video, an up-to-date
+		dual core processor from Intel or AMD running at at least 2.6 GHz is
 		necessary.</p>
 
-	<p>It is also necessary to have enough network bandwidth available
-		when streaming HD videos. It is nearly impossible to stream a full HD
-		video via WLAN, even when restricting network bandwidth and setting
+	<p>You should also have enough network bandwidth available when streaming HD
+		videos. It is nearly impossible to stream a full HD video via
+		wireless LAN, even when restricting network bandwidth and setting
 		video quality very low.</p>
 
 	<p>You should also have enough free RAM, which means about 512 MB.
-		There is also some space needed on your systems hard drive for
-		buffering videos. 500 MB free disk space is enough. If you want to
-		stream media files from an external hard drive please consider that an
+		There is also some space needed on your hard drive for
+		buffering videos. 500 MB of free disk space is enough. If you want to
+		stream media files from an external hard drive please consider that a
 		USB 1.1 connection is too slow for videos.</p>
```

Comment:
I've streamed videos from USB 2.0 nicely before so that bit seems accurate - it is only really USB 1x that usually wouldn't be capable of it